go.mod

```diff
@@ -5,19 +5,20 @@ go 1.13
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/coreos/prometheus-operator v0.38.0
-	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
-	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/openshift/library-go v0.0.0-20200402123743-4015ba624cae
-	github.com/stretchr/testify v1.4.0
-	golang.org/x/text v0.3.3 // indirect
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
+	github.com/go-logr/logr v0.2.1 // indirect
+	github.com/googleapis/gnostic v0.5.1 // indirect
+	github.com/openshift/api v0.0.0-20200916161728-83f0cb093902
+	github.com/openshift/client-go v0.0.0-20200827190008-3062137373b5
+	github.com/openshift/library-go v0.0.0-20200917093739-70fa806b210a
+	github.com/stretchr/testify v1.5.1
+	k8s.io/api v0.19.0
+	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Should we update this dependency?  It's very old and they have changed how the release versions on client-go.

Reply:
the one applying is this one https://github.com/openshift/cluster-autoscaler-operator/pull/166/files#diff-37aff102a57d3d7b797f152915a6dc16L23 v0.19.0